scripts/install-lansa-base.ps1

```diff
@@ -171,7 +170,11 @@ try
                 $downloaded = $true
             } catch {
                 $_
+<<<<<<< HEAD
                 $TotalFailedDownloadAttempts += 1
+=======
+                $TotalFailedDownloadAttempts += 1 
+>>>>>>> debug/paas
```

Comment:
This set of changes will not execute! You've left the merge choices in!
And you may need to rebase again if there is a conflict.